pandas/io/tests/test_parsers.py

```diff
@@ -210,6 +210,17 @@ def test_read_csv(self):
             # it works!
             read_csv(fname, index_col=0, parse_dates=True)
 
+    def float_precision_choices(self):
+        raise NotImplementedError
```

Comment:
OK.  (I was following the example of `ParserTests.read_csv() and ParserTests.read_table()`.)